commons-vfs2/src/test/java/org/apache/commons/vfs2/impl/VfsClassLoaderTests.java

```diff
@@ -263,14 +309,14 @@ public void rejectedExecution(Runnable r, ThreadPoolExecutor executor) {
             workQueue.addAll(rejected);
         }
         executor.shutdown();
-        executor.awaitTermination(30, TimeUnit.SECONDS);
+        executor.awaitTermination(5, TimeUnit.MINUTES);
```

Comment:
I don't think we want to tests to possibly take "forever" when they might fail anyway.

Reply:
Ah right, I was doing some very long tests (many many threads) to verify things.  I will knock this back down.